doc/sphinx/Pacemaker_Development/c.rst

```diff
@@ -15,6 +15,203 @@ Some existing Pacemaker code does not follow these guidelines, for historical
+| liblrmd       | lrmd    | lib/lrmd      | | include/crm/lrmd*.h     | .. index::                          |
+|               |         |               |                           |    single: C library; liblrmd       |
+|               |         |               |                           |    single: liblrmd                  |
+|               |         |               |                           |                                     |
+|               |         |               |                           | API for pacemaker-execd IPC         |
++---------------+---------+---------------+---------------------------+-------------------------------------+
+| libpacemaker  | pcmk    | lib/pacemaker | | include/pacemaker*.h    | .. index::                          |
+|               |         |               | | include/pcmki/*         |    single: C library; libpacemaker  |
+|               |         |               |                           |    single: libpacemaker             |
+|               |         |               |                           |                                     |
+|               |         |               |                           | High-level APIs equivalent to       |
+|               |         |               |                           | command-line tool capabilities      |
+|               |         |               |                           | (and high-level internal APIs)      |
++---------------+---------+---------------+---------------------------+-------------------------------------+
+| libpe_rules   | pe      | lib/pengine   | | include/crm/pengine/*   | .. index::                          |
+|               |         |               |                           |    single: C library; libpe_rules   |
+|               |         |               |                           |    single: libpe_rules              |
+|               |         |               |                           |                                     |
+|               |         |               |                           | Scheduler functionality related     |
+|               |         |               |                           | to evaluating rules                 |
++---------------+---------+---------------+---------------------------+-------------------------------------+
+| libpe_status  | pe      | lib/pengine   | | include/crm/pengine/*   | .. index::                          |
+|               |         |               |                           |    single: C library; libpe_status  |
+|               |         |               |                           |    single: libpe_status             |
+|               |         |               |                           |                                     |
+|               |         |               |                           | Low-level scheduler functionality   |
++---------------+---------+---------------+---------------------------+-------------------------------------+
```

Comment:
It might be worth explaining that the libpe_ ones are kind of arbitrary and confusing, and we are working towards merging them together.

Reply:
I thought about that, but quickly realized the document would double in length if I added all the "but we plan to ..."s. :) I'm not sure when we'll get around to it either, so I'd rather wait until it's at least being worked on.